src/services/completions.ts

```diff
@@ -5516,14 +5516,18 @@ function isModuleSpecifierMissingOrEmpty(specifier: ModuleReference | Expression
     return !tryCast(isExternalModuleReference(specifier) ? specifier.expression : specifier, isStringLiteralLike)?.text;
 }
 
-function getVariableOrParameterDeclaration(contextToken: Node | undefined) {
+function getVariableOrParameterDeclaration(contextToken: Node | undefined, location: Node) {
     if (!contextToken) return;

```

Comment:
Actually I didn't think about it when I opened this PR. Though for me its obvious that it can be `undefined` only when there is nothing behind the requesting position (as [essentially `contextToken` is `previousToken`](https://github.com/a-tarasyuk/typescript/blob/4c2bcc935563e509add094cc86ed5fb7eb82f51f/src/services/completions.ts#L4911)). Otherwise it will be something like OpenBraceToken.